Marlin/src/module/tool_change.cpp

```diff
@@ -918,7 +918,7 @@ void tool_change(const uint8_t new_tool, bool no_move/*=false*/) {
       DEBUG_ECHOLNPGM("No move (not homed)");
     }
 
-    TERN_(HAS_LCD_MENU, if (!no_move) ui.return_to_status());
+    TERN_(HAS_LCD_MENU, if (!no_move) ui.update());
```

Comment:
The purpose of this line is unclear, since tool change is not always related to LCD activities. Originally from 9bb8176d341.